protocol/xmrtaker/swap_state.go

```diff
@@ -669,7 +661,39 @@ func (s *swapState) lockAsset() (*ethtypes.Receipt, error) {
 		return nil, err
 	}
 
-	log.Infof("locked %s in swap contract, waiting for XMR to be locked", providedAmt.StandardSymbol())
+	log.Infof("locked %s in swap contract, waiting for XMR to be locked", s.providedAmount.StandardSymbol())
+	return receipt, nil
+}
+
+func (s *swapState) lockAndWaitForReceipt(
+	cmtXMRMaker, cmtXMRTaker [32]byte,
```

Comment:
Should we match these names to the names in the contract?
`pubKeyClaim` and `pubKeyRefund`

It's helpful that the above names have Maker and Taker in them, but I'm not sure everyone will know that `cmt` means commitment and what exactly the maker/taker committed to.

Reply:
it would make more sense to change the names in the contract, since what's stored is a hash of the swap public key, not the public key

Reply:
We could put the 2 values on separate lines and add a code comment after each one saying something like `commitment to public claim key` and `commitment to public refund key`.

Reply:
I think it's pretty well documented already in the swap contract and code-wise but can add if you think it's useful